contracts/BSCValidatorSet.sol

```diff
@@ -90,14 +98,18 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     uint256 enterMaintenanceHeight;     // the height from where the validator enters Maintenance
     bool isMaintaining;
 
+    // BEP-126 Fast Finality
+    bytes voteAddress;
+
     // reserve for future use
-    uint256[40] slots;
+    uint256[39] slots;
   }
 
   /*********************** cross chain package **************************/
   struct IbcValidatorSetPackage {
-    uint8  packageType;
+    uint8 packageType;
     Validator[] validatorSet;
+    ValidatorExtra[] _validatorExtraSet;
```

Comment:
Can we just use `Address[]` to save the gas?

Reply:
Fixed